tests/charm++/queue/Makefile

```diff
@@ -8,6 +8,8 @@ test: $(TARGETS)
 	$(call run, ./pgm  +p1)
 	$(call run, ./msgqtest  +p1)
 
+smptest: $(TARGETS)
```

Comment:
Should this have some run commands with it?

Reply:
These test is a sequential test that just creates a queue and does some tests on it. Does need to be run on multiple pes/cores/nodes/etc.